src/models/converters/tests/mod.rs

```diff
@@ -3,4 +3,5 @@ mod get_transfer_direction;
 mod transfers;
 mod transfer_erc20;
 mod transfer_erc721;
-mod transfer_ether;
\ No newline at end of file
+mod transfer_ether;
+pub (super) mod balances;
```

Comment:
when do we use `pub (super)`?

Reply:
For tests in general I would say, like you did for `missing_signers` I think it allows you to have visibility as if the tests were at the same crate the method you are testing (as long as you are only 1 crate deep, which I think is the case for tests)